app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class RecipeIngredientsController < ApplicationController
+  before_action :find_recipe
```

Comment:
Are you sure that we should call this action in case of `new`? I think that we shouldn't.